src/v2/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -420,6 +409,8 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
                     onSelect={value => onSelectSavedAddress(value)}
                     handleClickEdit={this.handleClickEdit}
                     inCollectorProfile={false}
+                    showModal={showModal}
```

Comment:
Related to my previous comment, I don't think this is actually used anywhere within SavedAddresses, and management of it here could be removed?

Reply:
In fact, I think that cleanup could go further -- `handleClickEdit` and `this.state.editAddressIndex` I don't think are used in this component now that the editing happens within `SavedAddresses`.